ocs_ci/utility/framework/fusion_fdf_init.py

```
@@ -48,15 +39,20 @@ def __init__(self, deployment_type: str) -> None:
 
         """
         try:
-            self.deployment_type = deployment_type
-            self.log_basename = LOG_NAMES[deployment_type]
-            self.run_id = generate_run_id()
+            log_basename = LOG_NAMES[deployment_type]
         except KeyError:
             raise InvalidDeploymentType(
                 f"Deployment type '{deployment_type}' is invalid. "
                 f"Please provide one of the following: {list(LOG_NAMES.keys())}"
             )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)